### ISSUE TITLE
Update to version v5.5.post202210041706

### ISSUE THREAD

**Issue by lrineau** [conda-forge/cgal-feedstock#74]
### Comment:

Now that https://github.com/CGAL/cgal-swig-bindings has releases, like https://github.com/CGAL/cgal-swig-bindings/releases/tag/v5.5.post202210041706, what should I do so that this feedstock get automatic updates like for the `cgal-cpp` feedstock (for example https://github.com/conda-forge/cgal-cpp-feedstock/pull/29)?

**Comment by chrisburr** [comment]
On the [conda-forge status page](https://conda-forge.org/status/) under "version updates" > "failed" it shows:
```
The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!

Please check the URLs in your recipe with version '5.5.1.post202210141207' to make sure they exist!

We also found the following errors:

 - recipe did not appear to change even though the bot said it should have
```

The issue is because the URL isn't defined using the version so the bot can't figure out what to do. I'll push something to #75 in a moment.

**Comment by lrineau** [comment]
Fixed by #80. Thanks again @xylar. That was an old issue.
